libgloss/patmos/close.c

```diff
@@ -33,11 +33,20 @@
 #undef errno
 extern int  errno;
 
+//******************************************************************************
+/// patmos-plug: close: stub called if 'patmosplug_close(int)' is not defined.
+int _patmosplug_close(int file) {
+  errno = EBADF;
+  return -1;
+}
+
+int patmosplug_close(int file)
+    __attribute__((weak, alias("_patmosplug_close")));
+
 //******************************************************************************
 /// _close - close a file descriptor.
 int _close(int file)
 {
   // TODO: implement for simulator target
```

Comment:
We would probably want to move this TODO into `_patmosplug_clone` with the code it originally was meant for.